source/_static/_templates/theme/sidebar.html

```diff
@@ -0,0 +1,22 @@
+<nav class="md-nav md-nav--primary" data-md-level="0">
+  <label class="md-nav__title md-nav__title--site" for="__drawer">
+    <a href="{{ pathto(master_doc)|e }}" title="{{ docstitle|striptags|e }}" class="md-nav__button md-logo">
+      {% if theme_logo_icon %}
+        <i class="md-icon">{{ theme_logo_icon }}</i>
+      {% else %}
+        <img src="{{ pathto('_static/' ~ logo, 1) }}" alt="{{ html_title }} logo" width="48" height="48">
+      {% endif %}
+    </a>
+    <a href="{{ pathto(master_doc)|e }}"
+       title="{{ docstitle|striptags|e }}">{{ theme_nav_title or shorttitle }}</a>
+  </label>
+  {%- if theme_repo_url %}
+    <div class="md-nav__source">
+      {% include "repo.html" %}
+    </div>
+  {% endif %}
+  {% include "globaltoc.html" %}
+  <label class="md-nav__title md-nav__title--site" for="__drawer">
+<a title="Download PDF Manual" onclick="_gaq.push(['b._trackEvent', 'Percona Server for MongoDB', 'Download', 'Download Manual Percona Server for MongoDB 4.4 ']);" href="https://learn.percona.com/download-percona-server-mongodb-4-4-manual" class="md-nav__link">Download PDF Manual <span class="glyphicon glyphicon-chevron-down"></span></a>
```

Comment:
That hard-coded version string might be difficult to maintain. I wonder if there's a way to replace it with a variable?